### PR TITLE
Creating A Custom Animated Shimmer Effect with Jetpack Compose

### LINKED ISSUE #5070: [Android] Creating an Custom Animated Shimmer Effect Jetpack Compose
#### Introduction
In this tutorial, I'm going to show the reader how to implement an Animated Shimmer effect in android applications using a modern UI toolkit Jetpack Compose. The shimmer effect gives the impression of extra light on the desired areas of the face. It makes the app look radiant and get a beautiful glow. Usually, the shimmer effect is used to show that the app is loading some data which maybe from the internet. When data is loaded, the shimmer effect stops and the user can get a view of the items that were being loaded.

#### Key takeaways 
- Implementing Shimmer Effect in Jetpack Compose
- Creating Shimmer Grid Items
- How to customize time, color, and Animate the Shimmer effect

#### Article quality
My topic is unique in that most of the written articles on the Jetpack Compose Shimmer Effect uses libraries. In this article, I will show the reader on how to create the Shimmer Effect by writing code only without using any library.

#### References
https://developer.android.com/jetpack/compose/animation

### PR DESCRIPTION
**Pull request by kirwa-elyjah**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5070

### CODE REVIEW

**Comment by hectorkambow** [comment]
@Ericgacoki  👋  
thank you for jumping on this PR 

Is there any older PR that you can review instead? I believe we have a few unassigned that have been open for a few days. 

This one was opened an hour ago. Thank you

**Comment by Ericgacoki** [comment]
> @Ericgacoki wave thank you for jumping on this PR
> 
> Is there any older PR that you can review instead? I believe we have a few unassigned that have been open for a few days.
> 
> This one was opened an hour ago. Thank you

@hectorkambow :wave: 
Thanks for pointing this out. I had glanced through the old PRs and none of them seemed too familiar to me or within my review areas. No worries though, let me take a second look and find one that I can follow along with 🤗

**Comment by Ericgacoki** [comment]
Greetings @hectorkambow :hand: 
I'll proceed with this one since it's in a reasonable position in the queue.

**Comment by Ericgacoki** [comment]
Hello @kirwa-elyjah, we're glad to see you at Section Eng-ed. Congrats on opening your first PR 👏 🎅🏿

File: content/authors/Kirwa-Elyjah/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Elyjah Kirwa
+type: authors
+github: https://github.com/kirwa-elyjah
+images:
+  - url: /engineering-education/authors/kirwa-elyjah/avatar.png
+---
```
Comment (Ericgacoki) [inline content/authors/Kirwa-Elyjah/index.md:7]: Please edit the author file name to match the one in this link.

**Comment by WanjaMIKE** [comment]
Plagiarism check is done.

**Comment by WanjaMIKE** [comment]
Nice work gents @kirwa-elyjah @Ericgacoki

**Comment by WanjaMIKE** [comment]
Merged #5753 into master

**Pull request merged**